python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
Surely it won't as you're subverting:

GangaList.append where GangaList is a GangaObject
to
GangaList._list.append where GangaList._list is a list object which is not governed by the ```__set__``` and ```__get__``` methods I think.